task7(28)/task7(28)/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task7_28_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите X");
+            x = double.Parse(Console.ReadLine());
+            Console.WriteLine(f(x));
+            Console.ReadKey();
+        }
+        static double f (double x)
+        {
+            double k;
+            if (x < 0)
+            {
+                k = 0;
+                return k;
+            }
+            else if (0 <= x && x <= 1)
+            {
+                k = Math.Sqrt(x);
+                return k;
+            }
+            else
+            {
+                k = 1;
+                return k;
+            }
```

Comment:
Уж если ввели переменную, то используйте с умом
```suggestion
            double k;
            if (x < 0)
                k = 0;
            else if (x <= 1)
                k = Math.Sqrt(x);
            else
                k = 1;
                
            return k;
```
А лучше без лишних переменных и в этом случае даже без `else`
```suggestion
            if (x < 0)
                return 0;
                
            if (x <= 1)
                return Math.Sqrt(x);
            
            return 1;
```